examples/react-app/SingleStream.jsx

```diff
@@ -24,12 +24,14 @@ export const SingleStream = () => {
   let vapixParams = {}
   try {
     vapixParams = JSON.parse(window.localStorage.getItem('vapix')) ?? {}
+    
   } catch (err) {
     console.warn('no stored VAPIX parameters: ', err)
   }
 
   useEffect(() => {
     authorize().then(() => {
+      fetchTransformationMatrix().then(console.log)
```

Comment:
Should probably not log the matrix. But perhaps use `debug` package function instead?

Reply:
Fixed